sirepo/job_supervisor.py

```diff
@@ -509,7 +514,11 @@ def _ops_to_cancel():
         ):
             # Valid, completed, transient simulation
             # Read this first https://github.com/radiasoft/sirepo/issues/2007
-            return await self._receive_api_runStatus(req)
+            r = await self._receive_api_runStatus(req)
+            if r.state == job.MISSING:
+                # happens when the run dir is deleted w/o us knowing (ex admin cli deletion)
```

Comment:
`ex pkcli.admin.purge_basic_users` so a reader can find it. `admin cli deletion` is a bit too cryptic.

Reply:
I don't see this change.